recipes/tsl_robin_map/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "tsl_robin_map" %}
+{% set version = "v0.5.0" %}
+{% set sha256 = " 220be06fa44005af67dc4b49fdfd96a66588b769eb53b0c899c6d8bf77033d84" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/Tessil/robin-map/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1000
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/tsl  # [unix]
+    - test -f ${PREFIX}/include/tsl/robin_map.h  # [unix]
+    - test -f ${PREFIX}/lib/cmake/tsl_robin_map/tsl_robin_mapConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/tsl_robin_map/tsl_robin_mapConfigVersion.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\tsl\robin_map.h (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\tsl_robin_mapConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\tsl_robin_mapConfigVersion.cmake (exit 0) else (exit 1)  # [win]
```

Comment:
This can be slightly simpler if you do a `if not ... exit 1` but I don't have a strong preference.